Makefile

```diff
@@ -31,8 +31,10 @@ all: build test
 # Downloads EMR packages. Skips if the tar file containing EMR packages has been made.
 
 init:
+	python --version
+	pip install --upgrade pip
 	# pipenv > 2022.4.8 fails to build smspark
-	python3 -m pip install pipenv==2022.4.8
+	python -m pip install pipenv==2022.4.8
```

Comment:
why do we remove python 3 from the command?

Reply:
python vs python3 etc really depends upon the local python executable in your env.